engines/tensorrt/build.gradle

```diff
@@ -31,14 +33,14 @@ processResources {
                 new URL("${url}/${entry.key}").withInputStream { i -> file.withOutputStream { it << i } }
             }
         }
-        file("${project.buildDir}/classes/java/main/native/lib").mkdirs()
+        file("${baseResourcePath}/native/lib").mkdirs()
         copy {
             from jnilibDir
```

Comment:
Did you verify whether changes to the files in this folder will trigger task execution?

Reply:
Same reasoning as previous - the files would not change if the properties do not